pandas/_libs/tslibs/nattype.pyi

```diff
@@ -19,6 +19,7 @@ class _NatComparison:
 
 class NaTType:
     value: np.int64
+    _value: np.int64
```

Comment:
did you add a .value property for NaT?

Reply:
good point, thanks - should it have the same value, right?

Reply:
NPY_NAT, yes

Reply:
cool, have added, thanks

any further comments? on board with the change, or should something else be used in place of `._value`?

Reply:
_value works for me